ehr2vec/effect_estimation/main_estimator.py

```
@@ -411,9 +424,30 @@ def _transform_column(column: pd.Series, delta: float) -> pd.Series:
         # Convert to logit space using scipy's stable implementation
-        logits = logit(column)
+        logits = logit(probas)
         # Add bias in logit space
-        biased_logits = logits + delta
+        biased_logits = logits + delta + sigmas
         # Convert back to probability space using scipy's stable implementation
         return expit(biased_logits)
+
+    @staticmethod
+    def _power_distortion(
+        probas: pd.Series, alpha: float = 1.0, constants: dict = {}
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)